phil/what-is-master.md

```diff
@@ -1,8 +1,37 @@
-# What is master?
+# What is the `/master` Branch?
 
 Currently, it is "skimmed-over code" (not necessarily tested).
 
-**Proposal:** It is tested code (at least lightly).
+**Proposal:** It is tested code (at least lightly by code reviewer or QA).
+
+**Note:** We _should_ begin using manifestly properly; this document is only about the label of the bottom row in [JP's diagram](https://www.dropbox.com/s/uwp0jq8luo51uqw/release_candidates_v2.pdf?dl=0). I think a deploy to any env should add an entry to [deploy-manifests](https://github.com/openstax/deploy-manifests).
```

Comment:
Re "a deploy to any env should add an entry"... this could be done but just to make sure we are on the same page it would have to be done outside of manifestly and to a different file (not the manifest file).  Right now, commit comments include information on when the manifest was deployed and to where, but in the (hopeful) case where manifests don't change from env to env (when there are no bugs), manifestly will not be able to re-push the manifest (git/manifestly won't allow an identical commit).  Nothing stopping us from maintaining other files in that repo that have other info like a list of manifests and timestamps per env.  And also possible manifestly could be modified to help with those other files.